lib/OpenQA/BuildResults.pm

```diff
@@ -106,8 +107,9 @@ sub compute_build_results {
         }
         $jr{reviewed_all_passed} = $jr{passed} == $count;
         $jr{total}               = $count;
-        $jr{reviewed}            = $jr{failed} > 0 && $jr{labeled} == $jr{failed};
-        $builds{$b}              = \%jr;
+        my $failed = $jr{failed} + $jr{softfailed};
```

Comment:
I understood it that way - as you described in [comment below](https://github.com/os-autoinst/openQA/pull/960#issuecomment-255685416). From my POV I just want to preserve softfails to be considered as passed.